index.js

```diff
@@ -143,8 +143,15 @@ HtmlWebpackPlugin.prototype.apply = function (compiler) {
           });
       })
       .then(function (html) {
-        // Add the stylesheets, scripts and so on to the resulting html
-        return self.postProcessHtml(html, assets);
+        // Prepare script and link tags
+        var assetTags = self.generateAssetTags(assets);
+        var pluginArgs = {head: assetTags.head, body: assetTags.body, plugin: self, outputName: self.childCompilationOutputName};
```

Comment:
Well that is already available in the plugin context - so you could do sth like:
```js
compiler.plugin('compilation', function(compilation) {
   compilation.plugin('html-webpack-plugin-alter-asset-tags', callback.bind(this, compilation));
}).bind(this);

function callback(compilation, htmlWebpackPluginData) {

}
```